src/content/en/fundamentals/instant-and-offline/_toc.yaml

```diff
@@ -3,4 +3,6 @@ toc:
     section:
     - title: The Offline Cookbook
       path: /web/fundamentals/instant-and-offline/offline-cookbook/
+    - title: Web Storage Overview
+      path: /web/fundamentals/instant-and-offline/web-storage-overview/
```

Comment:
We should rename this directory just `web-storage` since we'll want to put other docs in here at some point.